index.html

```diff
@@ -1,67 +1,140 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-	<title>Derek Graham</title>
-	    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Derek Graham</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="vendor/styles/normalize.css">
     <link rel="stylesheet" href="styles/icons.css">
     <link rel="stylesheet" href="styles/base.css">
     <link rel="stylesheet" href="styles/layout.css">
-    <link rel="stylesheet" href="styles/modules.css">
-</head>
-<body>
-    <header class="">
-      <section class="">
+<!--     <link rel="stylesheet" href="styles/modules.css">
+ -->    
+ 	<link rel="stylesheet" href="styles/modules/nav.css">
+ 	<link rel="stylesheet" href="styles/modules/about.css">
+ 	<link rel="stylesheet" href="styles/modules/projects.css">
 
-        <h1>Derek Graham</h1>
-        <h2>web developer &#x2022 maker</h2>
-      </section>
-      <hr>
-      <nav class="main-nav">
-        <ul>
-        	<li>About</li>
-<!--         	<li>Analog</li>
- -->        <li>Projects</li>
-        	<li>Skills</li>
-        	<li>Experience</li>
-        </ul>
-      </nav>
-    </header>
-    <section>
-    	<h4>About</h4>
-    	<hr>
-    	<div class="about">
-    	<img src="images/derek_2009_067.jpg">
-    		
-    	<p class="about tagline">I have been a maker my whole life.</p>
-    	<p class="about">I am  seeking new nails for my hammer as I chase ideas from the physical world to the virtual.</p>
-    	<p class="about">Currently learning Web Development at Codefellows.org - Coding 201 and 301.</p>
-    	<br>
-    	<p>Follow me on twitter.</p>
-    	<p>check out my projects on github</p>
-    	</div>
-    </section>
-    <section><h4>Projects</h4><hr>
+    <script id='project-template' type="text/x-handlebars-template">
+        <article class="template" data-category="{{category}}">
+            <header>
+                <h5>{{name}}</h5>
+                <div class="byline">
+                    By
+                    <address><a href="">{{maker}}</a></address>
+                    created
+                    <time pubdate={{publishedDate}} datetime="2000-01-01">Publish Time</time>
+                </div>
+            </header>
+            <section class="project-body">{{description}}</section>
+            <section class="project-image hidden">
+            <a class='project-image-link' href=""><img class='project-image' src="{{mediaURL}}"></a>
```

Comment:
Since you have a video, you should probably look into Handlebars if statements. http://handlebarsjs.com/builtin_helpers.html I think you might find them useful.